### PR TITLE
add cypress, made test on blog, contacts, portfolio, services

### PR DESCRIPTION
**Pull request by stoyanovk**

### CODE REVIEW

File: cypress/integration/contacts.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+describe('contacts page', function() {
+  it('form send', function() {
+    cy.visit('http://localhost:8000/contacts/');
```
Comment (mikhailnetkachev-halolab) [inline cypress/integration/contacts.spec.js]: Can we get it from `process` or not? Usually Gatsby starts on 8000, but this behavior can change.

File: cypress/integration/blog.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    cy.visit('http://localhost:8000/blog');
+    cy.contains('With Love from Halo Lab').scrollIntoView({
+      duration: 2000,
+    });
+  });
+  it('check filter', function() {
+    cy.contains('Our Blog').scrollIntoView({
+      duration: 1500,
+    });
+    cy.wait(1000);
```
Comment (lazio) [inline cypress/integration/blog.spec.js]: kill waits or run them only in debug mode

File: cypress/integration/contacts.spec.js (hunk 1 above)
Comment (lazio) [reply cypress/integration/contacts.spec.js]: can we have two commands
```
npm tun test // that will test localhost
npm run test:prod // that will test www.halo-lab.com
```
?

File: cypress/integration/portfolio.spec.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  it('Check page rendering', function() {
+    cy.visit('http://localhost:8000/portfolio/');
+    cy.contains('Design that inspires').scrollIntoView({ duration: 2000 });
+  });
+  it('check show more', function() {
+    cy.contains('Design that inspires').scrollIntoView({ duration: 1500 });
+    cy.wait(800);
+    cy.contains('INSPIRE MORE').click();
+    cy.wait(800);
+    cy.contains('INSPIRE MORE').click();
```
Comment (lazio) [inline cypress/integration/portfolio.spec.js]: check that there are more elements after click

File: cypress/integration/contacts.spec.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    cy.contains('Company')
+      .click()
+      .type('Stark Industries');
+    cy.contains('Email')
+      .click()
+      .type('tony_stark@gmail.com');
+    cy.contains('What is your project about?')
+      .click()
+      .type('Some charming message');
+    cy.contains('submit').click();
```
Comment (lazio) [inline cypress/integration/contacts.spec.js]: check that success message appears

File: cypress/integration/blog.spec.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    cy.contains('Agency').click();
+    cy.wait(1000);
+    cy.contains('Case Studies').click();
+    cy.wait(1000);
+    cy.contains('News').click();
+    cy.wait(1000);
+    cy.contains('Agency').click();
```
Comment (lazio) [inline cypress/integration/blog.spec.js]: we don't need to test every section, instead, please test one section + click on a particular post. 
after click you should check that title of the clicked element should be equal to title on a blog post page

File: cypress/fixtures/example.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (lazio) [inline cypress/fixtures/example.json]: can we remove this file?

**Pull request merged**